mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MessageMapper.java

```diff
@@ -173,6 +173,11 @@ long countUnseenMessagesInMailbox(Mailbox mailbox)
     Flags getApplicableFlag(Mailbox mailbox) throws MailboxException;
 
     /**
+     * Return the list of all uids of the {@link Mailbox}
```

Comment:
return a list containing all MessageUid of Messages that belongs to given Mailbox